webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx

```diff
@@ -457,14 +457,19 @@ const AdvanceTextEditor = ({
                         role='application'
                         id='advancedTextEditorCell'
                         data-a11y-sort-order='2'
-                        aria-label={Utils.localizeMessage(
-                            'channelView.login.successfull',
-                            'Login Successfull',
-                        ) + ' ' + ariaLabelMessageInput}
+                        aria-describedby='advancedTextEditorCellAriaLabel'
                         tabIndex={-1}
                         className='AdvancedTextEditor__cell a11y__region'
                     >
                         {labels}
+                        <label
+                            id='advancedTextEditorCellAriaLabel'
```

Comment:
These IDs need to be unique because there can actually be multiple `AdvancedTextEditor`s on screen at once: one in the centre channel for posts and one in the RHS for comments. `props.location` should be different between those two, so could you change this to 
```js
`advancedTextEditorLabel-${location}`
```
so that both of those have different IDs?

I noticed React now has a `useId` hook to use for this case, but unfortunately, we can't use it here because we're not on React 18 yet.